doc/adr/0013-flame-graph-support.md

```diff
@@ -22,7 +22,8 @@ There are 2 possibilites to support the flame graph in the TSP.
 
 The view itself is very similar to data providers of type `TIME_GRAPH`. The main difference is that the x-axis is the duration and not time. The Trace Compass back-end currently has such a data provider available with the same API than `TIME_GRAPH` data providers. However, this data provider is not exposed to the server through the `getDescriptor()` method of the corresponing data provider factory. It will need a new type to be added to the `ProviderType` enum.
 
-With this solution similar endpoints can be defined which have the advantage that the endpoints and data structures are known. Back-end filtering and highlighting would be supported out of the box. Assuming the new `Provider Type` is called `gantt` the following endpoints will be defined:
+With this solution the existing endpoint `TIME_GRAPH` endpoint and be agumented or similar endpoints can be defined which have the advantage that the endpoints and data structures are known. Back-end filtering and highlighting would be supported out of the box. 
```

Comment:
can be augmented?

do we want to indicate that the additions would include a parameter to indicate that the timegraph should be independent and not be synchronized with the time range of other views?

Reply:
Not sure if we need to say that here. In the end it's the FE decision to synchronize with the current zoom range or selection range or not at all. I understand that it might be beneficial that the BE provides this information for each data provider, though.